tests/go/GoReturnsBearTest.py

```diff
@@ -29,6 +29,5 @@
 
 GoReturnsBearTest = verify_local_bear(
     GoReturnsBear,
-    (good_file1,
-     good_file2,),
+    (good_file1, good_file2,),
```

Comment:
This was the first newcomer issue I solved...and I did all the stuff in master branch...since then this commit is there in this PR as well...any suggestions how can I remove this commit from this PR ?